app/ui/src/app/support/support.component.html

```diff
@@ -1,77 +1,90 @@
-            <div class="col-md-4">
+    <div class="panel panel-default support-panel">
+      <div class="panel-body">
+        <p>Select the integrations to capture diagnostics for:</p>
+        <form novalidate class="support-form">
+          <fieldset>
+            <div class="syn-form-input-list">
+              <div class="syn-form-input-list__option">
+                <label>
+                  <input type="radio"
+                        id="alllogs"
+                        name="logs" 
+                        [value]="true" 
+                        [(ngModel)]="allLogsSelected" 
+                        (click)="deselectAll()">
+                  All integrations 
+                </label>
+              </div>
+              <div class="syn-form-input-list__option">
+                <label>
+                  <input type="radio" 
+                        id="specificlogs" 
+                        name="logs" 
+                        [value]="false" 
+                        [(ngModel)]="allLogsSelected"> 
+                  Specific integrations
+                </label>
+              </div>
+              <!-- Toolbar -->
+              <div *ngIf="!allLogsSelected" class="container-fluid">
               <pfng-toolbar [config]="toolbarConfig"
                             (onFilterChange)="filterChanged($event)"
-                            (onSortChange)="sortChanged($event)">
-              </pfng-toolbar>
-            </div>
-            <div class="col-md-8 filter_labels">
-              <div>
-              <strong>{{ selectedItems() }} of {{ totalItems() }} Items</strong>
-              </div>
-            </div>
-          </div>
-          <div class="row">
-          <!-- List displayed here -->
-          <syndesis-loading >
-            <pfng-list [items]="items | objectPropertyFilter:filter | objectPropertySort:sort"
-                      [config]="listConfig"
-                      [itemTemplate]="listItemMain"
-                      (onSelectionChange)="handleSelectionChange($event)">
-            </pfng-list>
-          </syndesis-loading>
-
-          <!-- One list item, main row -->
-          <ng-template #listItemMain
-                      let-item="item">
-            <div class="list-pf-content list-pf-content-flex">
-              <div class="list-pf-content-wrapper">
-                <div class="list-pf-main-content">
-                  <div class="list-pf-title">
-                    {{ item.name }}
-                  </div>
-                  <div class="list-pf-description">
-                    <i>{{ item.description }}</i>
+                            (onSortChange)="sortChanged($event)"
+                            [viewTemplate]="viewTemplate">
+                <ng-template #viewTemplate>
+                  <div class="toolbar-pf-action-right">
+                    <div class="form-group">
+                      <strong>{{ selectedItems() }}</strong> of <strong>{{ totalItems() }}</strong> Items
```

Comment:
Two notes here, mostly addressed to @paoloantinori I guess:

Contrary to web docs rendered serverside, SPA documents are not evaluated upon rendering on screen for the first time but often through a change detection loop, which spawns a new cycle every few (very few) milliseconds. This means that whatever function call we drop in our views will be executed every few milliseconds too. And here we can see there're two function calls. While `totalItems()` just returns a class field `length` value, which is just fine, `selectedItems()`, on the contrary, performs a lambda call. That can become memory expensive in the long run, so my suggestions would be to have just a `selectedItems: number` field defined at the `SupportComponent` class and update it every time the `selected` property changes by means of the click handler, whatever it is.

Reply:
In any event, we can get away with these bindings as is, since they only affect this component, which is meant to be eventually destroyed. However, please take into consideration the principles exposed above for the future, thanks. :)